plugins/influxdb/influxdb.go

```diff
@@ -112,6 +112,11 @@ func (w *influxdbWorker) run() {
 }
 
 func buildPoint(m *pb.Metric) client.Point {
+	for k, v := range m.Labels {
+		if v == "" || k == "" {
+			delete(m.Labels, k)
```

Comment:
m is a pointer to a metric and there could be other goroutines looking at this metric at the same time. So unfortunately it isn't safe to modify m.Labels directly here. The slightly less-efficient but threadsafe version would be to copy all the non-empty labels into a new map.